components/PageFooter.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import Giscus from '@giscus/react';
+
+import styles from './styles.module.css'
+
+export const PageFooter: React.FC<{
+  isBlogPost: boolean
+}> = ({ isBlogPost }) => {
+  // only display comments and page actions on blog post pages
+  if (isBlogPost) {
+    return (
+      <div className={styles.comments}>
+        <Giscus
+          id="comments"
+          repo="giscus/giscus"
+          repoId="MDEwOlJlcG9zaXRvcnkzNTE5NTgwNTM="
+          category="General"
+          categoryId="MDE4OkRpc2N1c3Npb25DYXRlZ29yeTMyNzk2NTc1"
```

Comment:
Thank you for this PR, but please don't use this as the default, otherwise people who haven't configured this with their own repo will spam the giscus repo with their discussions.

Reply:
so, maybe just keep these configuration empty, then tell users how to set these in docs/wiki/readme?

Reply:
I haven't add configs to site.config.ts, there are too many values, so what's your suggestion?

Reply:
@laymonage already remove the giscus repo information, made these keys into config for giscus.